### ISSUE TITLE
How to keep GOW apps updated?

### ISSUE THREAD

**Issue by ABeltramo** [games-on-whales/wolf#24]
Wolf starts an app today and if the image is not present it'll pull; after that we have no way to automatically update images when we release updates in GOW.  
I think we should add it, and possibly add a way to configure it; ex: cron style cadency for updates.

**Comment by QuaiGoner** [comment]
Maybe we will not take on the path of always-updated images unattended for user, rather than provide a Web UI with a buttons for each image like that? (Screenshot from NetBoot.XYZ WebUI)
![image](https://github.com/games-on-whales/wolf/assets/41058180/7b893a24-94e4-42ad-ad55-936e52cbb517)

**Comment by ABeltramo** [comment]
Yeah, the web UI seems to always be a solution, but is it? In this case, I don't think it'll help much. You still have to manually check that page and without some kind of notification I don't think it'll be used much.

On the other side, there are already plenty of established ways to keep docker containers updated like Watchtower.

I think it might be useful, instead, to have an auto updater embedded into Wolf for the times when a user doesn't have that setup and just wants something that "just works". Obviously configurable so that it can be turned off by power users.

Just my 2 cents, I haven't really thought a lot about this..

**Comment by Mannshoch** [comment]
Would be an API an Idea? It would allow to do certain things and if a webui came in focus you could use the API for that.

**Comment by salty2011** [comment]
Hey as @ABeltramo mentioned this is something that a webui could do. Since this was last updated the WolfManager repo has started and this is a featured I plan on implementing

Adding this link to AppManagement feature and closing this
https://github.com/games-on-whales/wolfmanager/issues/27

This shouldnt bee too hard before the WolfManage code refactor did have something that sorta worked. the challenge was it could only provide an indicate of updates, no real easy way to tell how far behind in updates you were.

Closing as work will carry over to WolfManager
